src/app/components/PriceInfo.tsx

```diff
@@ -3,63 +3,83 @@ import { useAppSelector } from "../hooks";
 
 export function PriceInfo() {
   const priceInfo = useAppSelector((state) => state.priceInfo);
-  const lastPrice = priceInfo.lastPrice;
-  const change = priceInfo.change24h;
-  const high = priceInfo.high24h;
-  const low = priceInfo.low24h;
-  const volume = priceInfo.value24h;
+  const lastPrice = priceInfo.lastPrice.toFixed(3);
+  const change = priceInfo.change24h.toFixed(2);
+  const high = priceInfo.high24h.toFixed(3);
+  const low = priceInfo.low24h.toFixed(3);
+  const volume = priceInfo.value24h.toFixed(3);
```

Comment:
Maybe using the `displayAmount` function from utils instead of `.toFixed` here? It shortens very large numbers with K and M (like in order book).

Reply:
Nice, didn't know we had a utility class for this.